includes/templates/class-amp-post-template.php

```diff
@@ -322,7 +322,7 @@ private function build_post_comments_data() {
 	 */
 	private function build_post_content() {
 		$amp_content = new AMP_Content(
-			$this->post->post_content,
+			post_password_required( $this->post ) ? get_the_password_form( $this->post ) : $this->post->post_content,
```

Comment:
This is now somewhat like what [`get_the_content()`](https://developer.wordpress.org/reference/functions/get_the_content/) is doing:

```php
	// If post password required and it doesn't match the cookie.
	if ( post_password_required( $_post ) ) {
		return get_the_password_form( $_post );
	}
```

Reply:
Thanks for simplifying the password form logic, @westonruter.